### PR TITLE
Calling the Web API with Javascript and jQuery 3463

### LINKED ISSUE #3463: Add "Calling the Web API with Javascript and jQuery" to "Create a Web API with ASP.NET Core"
The legacy version of the [Intro to Web API](https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api) tutorial has a section [Calling the Web API with Javascript and jQuery](https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api#calling-the-web-api-with-javascript-and-jquery). Add the client side access section to the [ASP.NET Core WebAPI](https://docs.microsoft.com/en-us/aspnet/core/tutorials/#building-web-apis) tutorials.

Should the sample use jQuery?

Perhaps we should just point to #3261 
@danroth27  please review.

### PR DESCRIPTION
**Pull request by isaac2004**
Fixes #3463 

Added Include demoing how to query Web Api from clientside

@Rick-Anderson

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
[Review URLs](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C10%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201804100249074790-5914%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8mPDm%2FN9C6gH4KMKAyxdRq9uNTu19MN1aVvI2tHI%2BA4%3D&st=2018-04-10T02%3A47%3A35Z&se=2018-05-11T02%3A52%3A35Z&sp=r)

**Comment by Rick-Anderson** [comment]
@isaac2004 can you test the new code with the download code?

File: aspnetcore/includes/add-javascript-jquery/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[!code-html[Main](samples/sample3.html)]
+
+There are several ways to get jQuery. For this example, the [Microsoft Ajax CDN](/aspnet/ajax/cdn/overview) is used. You can also download it from [http://jquery.com/](http://jquery.com/).
+
+### Getting a List of Products
+
+To get a list of products, send an HTTP GET request to &quot;/api/products&quot;.
+
+The jQuery [getJSON](http://api.jquery.com/jQuery.getJSON/) function sends an AJAX request. For response contains array of JSON objects. The `done` function specifies a callback that is called if the request succeeds. In the callback, we update the DOM with the product information.
```
Comment (Rick-Anderson) [inline aspnetcore/includes/add-javascript-jquery/index.md]: > For response contains array of JSON objects. 

That's in the original. I don't understand it.

**Comment by Rick-Anderson** [comment]
@isaac2004  delete images. They don't work in the VS for Mac and the VS code versions.

Could we use a current jQuery version?

**Comment by isaac2004** [comment]
@Rick-Anderson updated jQuery Version, cleaned up comment and fixed examples to use ToDo instead of products

File: aspnetcore/includes/add-javascript-jquery/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Calling the Web API with Javascript and jQuery
```
Comment (scottaddie) [inline aspnetcore/includes/add-javascript-jquery/index.md]: Calling --> Call

File: aspnetcore/includes/add-javascript-jquery/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+In this section, an HTML page is added that uses AJAX to call the web API. jQuery is used to make the AJAX calls and to update the page with the results. Add an HTML page with the following:
+
+[!code-html[Main](samples/sample3.html)]
+
+There are several ways to get jQuery. For this example, the library is loaded from [http://jquery.com/](http://jquery.com/).
+
+### Getting a List of ToDos
```
Comment (scottaddie) [inline aspnetcore/includes/add-javascript-jquery/index.md]: - Getting --> Get
- List --> list

File: aspnetcore/includes/add-javascript-jquery/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+To get a list of ToDos, send an HTTP GET request to &quot;/api/todo&quot;.
+
+The jQuery [getJSON](http://api.jquery.com/jQuery.getJSON/) function sends an AJAX request to the Api, which returns a JSON structured item, representing an object or array. The `done` function specifies a callback that is called if the request succeeds. In the callback, we update the DOM with the Todo information.
+
+[!code-html[Main](samples/sample4.html)]
+
+### Getting a ToDo By ID
```
Comment (scottaddie) [inline aspnetcore/includes/add-javascript-jquery/index.md]: - Getting --> Get
- By --> by

**Comment by Rick-Anderson** [comment]
@scottaddie  it's probably easier to fix those yourself and do a commit. Are you OK with these changes without anyone testing the code?

**Comment by scottaddie** [comment]
Thank you for your help, @isaac2004!

**Comment by isaac2004** [comment]
Thanks for cleaning up my mess @scottaddie @Rick-Anderson !!!!

**Pull request merged**